src/Infiltrator.jl

```diff
@@ -47,9 +52,13 @@ and then this macro will serve as a "conditinal breakpoint", which starts inspec
 when its condition is `true`.
 """
 macro infiltrate(cond = true)
+  if __module__ === Core.Compiler && !isdefined(Core.Compiler, :Dict)
+    # XXX Dict isn't available in Core.Compiler, so make it available now
+    Core.eval(Core.Compiler, :(using .Main: Dict))
```

Comment:
Why's this necessary?

Reply:
`Core.Compiler` tries to be as small as possible, so it doesn't define `Dict`, so the evaluation of `:locals` would fail in `Core.Compiler` context.
Nevertheless when we use Infiltrator (after bootstrapping), we can make it available this way.